terraform/user-data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Why is this script called "user-data" when what it does seems to be install java-8?

Reply:
And what calls the script?

Reply:
All is explained here 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts

And currently, it is called in the ec2-*.tf files